numpy/lib/ufunclike.py

```diff
@@ -170,8 +210,4 @@ def isneginf(x, y=None):
     array([1, 0, 0])
 
     """
-    if y is None:
-        x = nx.asarray(x)
-        y = nx.empty(x.shape, dtype=nx.bool_)
-    nx.logical_and(nx.isinf(x), nx.signbit(x), y)
-    return y
+    return nx.logical_and(nx.isinf(x), nx.signbit(x), out)
```

Comment:
For a little more efficiency, we could change this to:

    nx.logical_and(nx.isinf(x, out), nx.signbit(x), out)

But that seemed out of scope for the bugfix/deprecation